.github/workflows/formatting.yml

```diff
@@ -27,6 +27,7 @@ jobs:
 
       - name: Install dependencies
         run: |
+          cd job_creator
```

Comment:
Does this mena black will only format the job creator since there is never a cd back up a level?

Reply:
No the terminal session does not get maintained between actions unless you tell it to.